#  Name the experiment. This name will be the name of the report
experiment_name: "param_guide"

#Save directory of results
results_directory: ''

#  Add the world file to benchmarking_tool/world and provide the name of the new file
world_name: 'scenario_world.world'

#  Add the map files to benchmarking_tool/map and provide the name of the new yaml fi
map_name: 'map.yaml' 

#  Add gazebo model path, if multiple path, please spereate them by a :
models_path: ''

#  Choose one of the following controllers, by adding its name to the controller type list
#  Note that these are the avaible controllers for the provided example
#  DWB
#  RPP
#  RPP_RSC
#  DWB_RSC
controller_type: ['DWB_RSC','RPP_RSC','DWB', 'RPP']   #name of behaviour tree 

#  This option is intended for testing the same controller several times
#  if the user do not need this test, set it to zero 
trails_num: 10

#  Add the absolute path of the robot navigation configuration
nav_config: '/home/riotu/ros2_ws/src/ros2_nav_benchmarking-main/turtlebot3/turtlebot3_navigation2/param/waffle.yaml'

#  Add the directory of the behaviour tree
behaviour_tree_directory: "/home/riotu/ros2_ws/src/ros2_nav_benchmarking-main/turtlebot3/turtlebot3_navigation2/param"

#  Add the absolute path of the urdf and model files inside single qoutes ''
urdf_file: '/home/riotu/ros2_ws/src/ros2_nav_benchmarking-main/turtlebot3/turtlebot3_description/urdf/turtlebot3_waffle.urdf'

#  If the robot is spawned by only urdf, then set the model_file to 'None'
model_file: '/home/riotu/ros2_ws/src/ros2_nav_benchmarking-main/turtlebot3_simulations/turtlebot3_gazebo/models/turtlebot3_waffle/model.sdf'

#  Provide the pose in a decimal form
spawn_pose_x: 4.0
spawn_pose_y: -3.0
spawn_pose_yaw: -3.13   #radians 

#  Specify either to send one goal, specific trajectory, or several waypoints 
#  square, circle, several_waypoints, one_goal
trajectory_type: 'circle'
#  One goal parameter 
goal_pose_x: 4.0
goal_pose_y: -3.0
goal_pose_yaw: -3.13   #radians 
#  Circule parameter, note that the place of spawn will be the center of the circle 
radius: 1.1
#  Square parameter 
side_length: 3.0
#  Specify way points in this formate [[x0,y0,yaw0_radians], [x1,y1,yaw1_radians], ...]
waypoints: [[-1.3,2.3,-3.13], [6.0,2.0,-3.13], [-4.0,4.0,-3.13]]

